linter/lints/crl/lints_test.go

```diff
@@ -251,3 +251,40 @@ func TestHasMozReasonCodes(t *testing.T) {
 	test.AssertEquals(t, res.Status, lint.Error)
 	test.AssertContains(t, res.Details, "MUST NOT include reasonCodes other than")
 }
+
+func TestHasValidTimestamps(t *testing.T) {
```

Comment:
Should have test cases for timezones, to ensure that `YYMMDDHHMMSZTTTT` gets rejected as non-UTC.

Ooh, a test case with `YYMMDDHHMMSSZ0000` would be a good test case, because I think the stdlib might interpret that as having a Location of UTC, but it's still a violation of our requirements here.

Also, I think it would be cool to have a testcase whose timestamp is formatted `YYMMDDHHMMZTT` (i.e. _missing seconds_, but with a malformed timezone that causes the length to be correct) to ensure that we catch that by having it fail to parse.

Reply:
Love these ideas, thank you!